<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScene</class>
 <widget class="QDialog" name="LoginScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1951</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_image2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../build-Final_Project-Desktop_Qt_5_12_12_MinGW_64_bit-Debug/LogIn.PNG</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_mail2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>610</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pass2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>690</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mail2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>620</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit2">
   <property name="geometry">
    <rect>
     <x>1780</x>
     <y>20</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>850</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
